package.json

```diff
@@ -71,7 +71,7 @@
     "mocha-lcov-reporter": "^1.3.0",
     "nock": "^9.0.13",
     "ps-node": "^0.1.5",
-    "semantic-release": "^7.0.1",
+    "semantic-release": "^7.0.2",
```

Comment:
This has been already done in https://github.com/apiaryio/dredd/pull/871/files, but thanks!